kotlin-extensions/src/main/kotlin/kotlinx/js/JsSet.kt

```diff
@@ -0,0 +1,14 @@
+package kotlinx.js
+
+@JsName("Set")
+external class JsSet<T> {
+    val size: Int
+        get() = definedExternally
+    fun add(value: T): JsSet<T>
+    fun clear()
+    fun delete(value: T)
+    fun entries(): JsIterator<T>
```

Comment:
FYI
1. In [TS](https://github.com/JetBrains/kotlin-wrappers/blob/4eab763236faf248f2d5eabf0aba4653e0903685/kotlin-typescript/src/main/generated/typescript/ReadonlySet.kt#L17)
2. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/entries)

Reply:
Sorry (-_-; )